### ISSUE TITLE
Missing frequency from GWAS

### ISSUE THREAD

**Issue by ariadnacilleros** [bulik/ldsc#424]
Hi, 

We are currently running a partitioned heritability analysis with our annotation and 10 GWAS. Some of them don't include the frequency of the SNP, and even though I have observed that this information is not among the essential columns required from a GWAS, we would like to know whether including it or not could impact our results. 
Thank you so much for this package and the detailed Wiki. 

Best, 

Ariadna Cilleros

**Comment by aksarkar** [comment]
@ariadnacilleros No, ldsc does not use the MAF information in the GWAS, only MAF information derived from the reference panel used to estimate LD scores.
